tests/test_flow/test_persistence_fuzz.py

```diff
@@ -270,17 +311,30 @@ def run(self, n_iterations):
                     self.model.expected_entity_value(affected_name)
                 )
 
-            if make_func_change or self._versioning_mode == 'auto':
-                expected_called_names = self.model.entity_names(
+            if make_func_change:
+                entity_names_in_between = self.model.entity_names(
```

Comment:
nit: why not just do `entity_names_in_between  = set(self.model.entity_names(...))`?

Reply:
I don't know why I didn't do that!